frontend/src/app/patients/PatientForm.tsx

```diff
@@ -121,38 +122,31 @@ const UPDATE_PATIENT = gql`
   }
 `;
 
-const Fieldset = (props) => (
-  <fieldset className="prime-fieldset">
-    <legend>
-      <h3>{props.legend}</h3>
-    </legend>
-    {props.children}
-  </fieldset>
-);
+interface Props {
+  activeFacilityId: string;
+  patientId?: string;
+  patient?: any; //TODO: TYPES
+}
 
-// TODO: move to an Address component
-const stateChoices = [
-  { label: "- Select - ", value: "" },
-  ...stateCodes.map((c) => ({ label: c, value: c })),
-];
-
-const PatientForm = (props) => {
+const PatientForm = (props: Props) => {
   const appInsights = useAppInsightsContext();
-  const trackAddPatient = useTrackEvent(appInsights, "Add Patient");
-  const trackUpdatePatient = useTrackEvent(appInsights, "Update Patient");
+  const trackAddPatient = useTrackEvent(appInsights, "Add Patient", {});
```

Comment:
The typedefs for these seem to require the extra arg but don't seem to blow up if it's missing.